tests/gear3/gear3_helloworld.nif

```diff
@@ -4,11 +4,12 @@
   (var :x . . (i +32) +12)
   (var :y . . (i +32) +12)
 
+  (import mod1)
```

Comment:
Not required in gear3! You can simply use `foo.c12.mod1` later and then it is picked up from `mod1`.

Reply:
I see. I added code to skip `import` because the result from `nifler parse` retains the `import mod1`.

Done